src/cgutils.cpp

```diff
@@ -2156,25 +2156,10 @@ static Value *emit_allocobj(jl_codectx_t &ctx, size_t static_size, Value *jt)
 {
     JL_FEAT_REQUIRE(ctx, dynamic_alloc);
     JL_FEAT_REQUIRE(ctx, runtime);
```

Comment:
Hmm I hadn't considered that, as it worked with `_dump_function` but seems to mess up the JIT indeed. Then again, I only use that feature through `_dump_function`...
Maybe a "used features mask" to be checked after lowering? Let's just leave it at what it is now and put that in a different PR.